test/connectivity/modules/NetworkController/NetworkInterfaceOfflineController.cs

```diff
@@ -23,73 +21,29 @@ public NetworkInterfaceOfflineController(string dockerInterfaceName)
 
             if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
```

Comment:
Please put todo here; dockerInterfaceName should not be required when running on Windows.  We need to refactor to allow instantiate different IController implementation.